ibmcloud/image/build.sh

```diff
@@ -224,19 +233,27 @@ echo "targetblocksize=512"
 echo "targetoffset=$target_offset"
 END
         chmod 755 "$helper"
-
-        mount -t proc proc "$dst_mnt/proc"
-        mount --bind /dev "$dst_mnt/dev"
-
         chroot "$dst_mnt" zipl -V
-
-        umount "$dst_mnt/proc"
-        umount "$dst_mnt/dev"
         rm "$helper"
         ;;
+    *)
+        echo -e "\nUpdating GRUB settings\n"
+        sed -i -r -e 's|^GRUB_CMDLINE_LINUX=|#\0"|' "$dst_mnt/etc/default/grub"
+        cat <<END >> "$dst_mnt/etc/default/grub"
+
```

Comment:
Is there special reason leave empty line in `/etc/default/grub` while not in `zip`l?

Reply:
In the case of zipl, we create a new temporary file.

For grub, we need to append several lines to `/etc/default/grub`, so I leave an empty line as a separator.